user_guide/feature_contexts/hooking_into_the_test_process.rst

```diff
@@ -160,33 +162,39 @@ There are two suite hook types available:
 * ``@BeforeSuite`` - executed before any feature runs.
 * ``@AfterSuite`` - executed after all features have run.
 
-Feature Hooks
--------------
+Tagged Hooks
```

Comment:
Ok, but same like an issue somewhere above, this section is literally copied/pasted from the [original content](https://raw.githubusercontent.com/Behat/docs/v3.0/guides/3.hooks.rst). If we need to modify the original content, I suggest to create a separate issue for that.